### ISSUE TITLE
New releases

### ISSUE THREAD

**Issue by sebhoerl** [eqasim-org/ile-de-france#188]
We should create a new release. Recently, we have worked quite a bit on the file structure which deserves to increase the version number. We should just wait one or two weeks though to see if all of this works well (see #185 for instance).

After, it might make sense to think of a release schedule, so we know when and why we want to make new releases. So far, we simply added things to the changelog and just had a long chain of development versions. Which is also ok, but some people may want to work with stable versions, and it will help in projects and papers to reference the version.

**Comment by Nitnelav** [comment]
Yes ! We should release more. 

I think the current scheme (semantic versioning) is ok, the use of the CHANGELOG is also very clear. 

We can do something like :

Keep updating the changelog the same way and *every month*, look at it:

- If it is empty (since last release), do nothing
- If it contains only "chores", increase patch number (1.3.1 -> 1.3.2)
- If it contains at least on "feat", increase minor number (1.3.1 -> 1.4.0)
- If it contains a big breaking change, increase major number (1.3.1 -> 2.0.0)

We'll probably see the major update coming long before it appears in the changelog. Maybe systematically adding the vehicles is a good candidate to major update...

What do you think ?

**Comment by sebhoerl** [comment]
Yes, we can just increase the major number after all these changes.

Regarding the changelog, I saw that tellae use some auto-generator based on the PRs. Maybe we could also install this and then have a clean process starting from 2.0.0. @vincent-leblond @leo-desbureaux-tellae  @MarieMcLaurent You have some advice on that?

**Comment by Nitnelav** [comment]
https://github.com/eqasim-org/ile-de-france/issues/276
